frontend/src/scenes/notebooks/Nodes/NotebookNodeReplayTimestamp.tsx

```diff
@@ -85,6 +77,7 @@ export const NotebookNodeReplayTimestamp = Node.create({
         return {
             playbackTime: { default: null, keepOnSplit: false },
             sessionRecordingId: { default: null, keepOnSplit: true, isRequired: true },
+            sourceNodeId: { default: null, keepOnSplit: true },
```

Comment:
Not one for now but maybe this should be a default attribute so all nodes can use it